Userland/Applications/PixelPaint/ImageEditor.cpp

```diff
@@ -337,29 +337,44 @@ GUI::MouseEvent ImageEditor::event_adjusted_for_layer(GUI::MouseEvent const& eve
     };
 }
 
-void ImageEditor::set_editor_color_to_color_at_mouse_position(GUI::MouseEvent const& event, bool sample_all_layers = false)
+Optional<Color> ImageEditor::color_from_position(Gfx::IntPoint position, bool sample_all_layers)
```

Comment:
nice name change! 👍

i feel that you could change that in other places too, of course leave “mouse” only when you deal with mouse events as parameters. 😝

Reply:
Hmm, I didn't change the name of this API.